modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_claim_service_base.rb

```diff
@@ -13,9 +13,15 @@ class DisabilityCompensationClaimServiceBase
       include Sidekiq::MonitoredWorker
 
       NO_RETRY_ERROR_CODES = ['form526.submit.noRetryError', 'form526.InProcess'].freeze
-
       LOG_TAG = '526_v2_claim_service_base'
 
+      sidekiq_retries_exhausted do |message|
+        ClaimsApi::Logger.log('claims_api_retries_exhausted',
+                              claim_id: message['args'].first,
+                              detail: "Job retries exhausted for #{message['class']}",
+                              error: message['error_message'])
+      end
+
```

Comment:
should we not consolidate / inherit from the service_base... we have a lot of duplicate logic that could be removed from here if we did. This file would shrink significantly.

Reply:
ahhh. started to respond earlier and keep getting pulled in to calls - sorry. also, you may have tagged the wrong person 😆 

I would say we're safe to remove these, right? We should probably record an error in the rescue in this claim, and then we could cut out the `log_exception_to_claim_record` we're really just adding the error message of "job retries exhausted" but our logging and the alert we'll be building should notify us of this. I do think it's worth adding the "update error  + `.save`" functionality to the claim though.

Also, we get the logs and details in DataDog now, I think we can start cutting out Sentry now in this particular flow. We're going to alert on these from DataDog, so we'll have access to find the issues / errors in argo or DataDog, so let's remove these special logs, unless you have a strong passion to keep sentry going. I'm not sure we have any commitment to continuing to use sentry through the program, so I would think our logging and DD handling is acceptable.

Reply:
yes, I have removed the entire file now, I just forgot to do that before putting it in review